docs/other/troubleshooting-gratia.md

```diff
@@ -260,7 +261,12 @@ Not much is printed to the screen, but you can see progress in the Gratia log fi
 
 
 !!! note 
-    Condor rotates history files, so you can only report what Condor has kept. Controlling the Condor history is documented in the Condor manual. In particular, see the options for [MAX_HISTORY_LOG](https://research.cs.wisc.edu/htcondor/manual/v8.6/3_5Configuration_Macros.html#21780) and [MAX_HISTORY_ROTATIONS](https://research.cs.wisc.edu/htcondor/manual/v8.6/3_5Configuration_Macros.html#21785).
+    Condor rotates history files, so you can only report what Condor has kept.
+    Controlling the Condor history is documented in the Condor manual.
+    In particular, see the options for
+    [MAX_HISTORY_LOG](https://htcondor.readthedocs.io/en/latest/admin-manual/configuration-macros.html#MAX_HISTORY_ROTATIONS)
+    and
+    [MAX_HISTORY_ROTATIONS](https://htcondor.readthedocs.io/en/latest/admin-manual/configuration-macros.html#MAX_HISTORY_ROTATIONS).
```

Comment:
This looks good but I'm not sure if this section is all that relevant anymore since we package the appropriate config by default:

- https://github.com/opensciencegrid/gratia-probe/blob/master/condor/99_gratia.conf
- https://github.com/opensciencegrid/gratia-probe/blob/master/rpm/gratia-probe.spec#L469

Reply:
This entire subsection (https://github.com/opensciencegrid/docs/blob/9106411e5fa17f067ee6122534b383144994b4b3/docs/other/troubleshooting-gratia.md#if-you-accidentally-did-not-set-per_job_history_dir-see-above) that this note lives in.

Reply:
Ok.  Not sure if it might still be relevant for non-rpm installs?  Anyway, dropping the section.

Reply:
> Not sure if it might still be relevant for non-rpm installs?

The thought of this for Gratia probe frightens me!